xl/trax/track.py

```diff
@@ -654,6 +654,9 @@ def get_tag_sort(
         elif tag == '__basename':
             # TODO: Check if unicode() is required
             value = self.get_basename()
+        elif tag == '__rating':
+            # TODO: Check if unicode() is required
```

Comment:
Can you remove this comment? It was probably copied from above case, but it is not applicable in the case of ratings (as we're dealing with integers).